src/options.c

```diff
@@ -474,7 +465,7 @@ void optionsParse(int argc, char *argv[])
             optionsParseStack(optarg);
             break;
         case 'C':
-            optionsParseWindowClassName(optarg);
+            opt.windowClassName = optarg;
```

Comment:
Could you make `opt.windowClassName` const in options.h ? Assuming there won't be any issues.